### PR TITLE
Aims 405 create batches and new bin

### PR DESCRIPTION
**Pull request by afstanton**
This PR is a cumulative branch of all work up until now. AIMS-403 is not contained in this, it is not yet started.

### CODE REVIEW

File: app/controllers/dispositions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+class DispositionsController < ApplicationController
```
Comment (jgondron) [inline app/controllers/dispositions_controller.rb]: Needs specs

File: app/services/build_deaccessioning_request.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class BuildDeaccessioningRequest < GetRequests
+  def self.call(item_id)
```
Comment (jgondron) [inline app/services/build_deaccessioning_request.rb]: Needs specs

File: app/services/is_bin_barcode.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module IsBinBarcode
-  PREFIX = "(BIN-)(ILL-LOAN|ILL-SCAN|ALEPH-LOAN)(-)".freeze
+  PREFIX = "(BIN-)(ILL-LOAN|ILL-SCAN|ALEPH-LOAN|DEAC-STOCK)(-)".freeze
```
Comment (jgondron) [inline app/services/is_bin_barcode.rb]: Needs specs

**Comment by afstanton** [comment]
Absolutely! I will cover all of these things over this coming weekend. Thank you for reviewing these!

**Comment by afstanton** [comment]
It just dawned on me to use the output of simplecov to see precisely where you were getting these. Took me long enough! :D

**Pull request merged**